test/XRP/fakes/fake-grpc-error.ts

```diff
@@ -0,0 +1,19 @@
+import grpc from 'grpc'
+
+/**
+ * Convenience class for creating more specific Error objects to mimick gRPC error responses.
+ */
+export default class FakeGRPCError extends Error {
+  /**
```

Comment:
I copied this class into /ILP to test my error handling.  Whoever merges second should consider putting this in a common location for all tests to use

Reply:
+1.

Either do this now or add a TODO and assign yourself to do it in a separate PR.